tokio/tests/rt_basic.rs

```diff
@@ -63,65 +63,6 @@ fn acquire_mutex_in_drop() {
     drop(rt);
 }
 
-#[test]
```

Comment:
This test is moved into `rt_common.rs` so that it is run against both schedulers.